docs/HowTo/Configure/Consensus-Protocols/IBFT.md

```diff
@@ -43,164 +51,198 @@ To use IBFT 2.0 requires an IBFT 2.0 genesis file. The genesis file defines prop
+scheme used in Ethereum. To generate the `extraData` RLP string for inclusion in the genesis file,
+use the [`rlp encode`](../../../Reference/CLI/CLI-Subcommands.md#rlp) Besu subcommand.
 
-The `extraData` property is RLP encoded. RLP encoding is a space efficient object serialization scheme 
-used in Ethereum. Use the Besu subcommand [`rlp encode`](../../../Reference/CLI/CLI-Subcommands.md#rlp) 
-to generate the `extraData` RLP string to include in the genesis file. 
+!!! example
 
-!!! example                                        
     ```bash
     besu rlp encode --from=toEncode.json
-    ```    
+    ```
 
-Where the `toEncode.json` file contains a list of the initial validators in ascending order. Use the Besu subcommand [`public-key export-address`](../../../Reference/CLI/CLI-Subcommands.md#export-address) to write the validator address and copy it to the `toEncode.json` file. For example:
+Where the `toEncode.json` file contains a list of the initial validators, in ascending order. To
+write the validator address and copy it to the `toEncode.json` file, use the
+[`public-key export-address`](../../../Reference/CLI/CLI-Subcommands.md#export-address) Besu
+subcommand. For example:
 
 !!! example "One Initial Validator in `toEncode.json` file"
+
     ```json
     [
      "9811ebc35d7b06b3fa8dc5809a1f9c52751e1deb"
     ]
-    ``` 
+    ```
 
-Copy the RLP encoded data to the `extraData` property in the genesis file. 
+Copy the RLP encoded data to the `extraData` property in the genesis file.
 
-### Block time 
+### Block time
 
-When a new chain head is received, the block time (`blockperiodseconds`) and round timeout (`requesttimeoutseconds`) 
-timers are started. When `blockperiodseconds` is reached, a new block is proposed. 
+When the protocol receives a new chain head, the block time (`blockperiodseconds`) and round
+timeout (`requesttimeoutseconds`) timers start. When `blockperiodseconds` expires, the protocol
+proposes a new block.
 
-If `requesttimeoutseconds` is reached before the proposed block is added, a round change occurs, and the block time and 
-timeout timers are reset. The timeout period for the new round is two times `requesttimeoutseconds`. The 
-timeout period continues to double each time a round fails to add a block. 
+If `requesttimeoutseconds` expires before adding the proposed block, a round change occurs, with
+the block time and timeout timers reset. The timeout period for the new round is two times
+`requesttimeoutseconds`. The timeout period continues to double each time a round fails to add a
+block.
 
-Generally, the proposed block is added before reaching `requesttimeoutseconds`. A new round is then started, 
-and the block time and round timeout timers are reset. When `blockperiodseconds` is reached, the next new block is proposed. 
+Usually, the protocol adds the proposed block before reaching `requesttimeoutseconds`. This causes
```

Comment:
Generally we try and avoid using `this` -> it adds to the cognitive load of the reader because they have to go back to the previous sentence to figure out what `this` is referring to. It's harder for ESL readers and translation as well.

Reply:
Fixed. The new para reads as follows:
"Usually, the protocol adds the proposed block before reaching \`requesttimeoutseconds\`. A new round then starts, resetting the block time and round timeout timers. When \`blockperiodseconds\` expires, the protocol proposes the next new block."